.circleci/config.yml

```
@@ -134,7 +134,7 @@ jobs:
       - run: npm run build
       - run: PYTHONPATH=. python scripts/nightly/get_version.py > __init__.py
       - run: mv __init__.py wagtail/__init__.py
-      - run: python setup.py bdist_wheel
+      - run: python pyproject.toml bdist_wheel
```

The command should be. With `pip install build` around line 83

```suggestion
      - run: python -Im build bdist_wheel
```

---

Thanks! Is `-I` necessary? I've never used it before

---

`-I` is for immutable. not necessary per se. `python -m build` would do